cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/AbstractReadyResourceOperator.java

```diff
@@ -103,6 +110,9 @@ public void handle(Long timerId) {
      */
     public boolean isReady(String namespace, String name) {
         R resourceOp = operation().inNamespace(namespace).withName(name);
+        if (resourceOp == null) {
```

Comment:
Did you actually see this being null? Why do we need this now if we didn't before?

Reply:
Yes, while running tests I got some warnings of dereferencing null. It also happens at master.